site/docs/guidelines/workflow.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Workflow
+navTitle: Workflow
+summaryParagraph: A workflow is a long task broken up into a series of smaller, linked steps.
+tags: ["Wizard", "Multi-step workflow", "Onboarding"]
+needToKnow:
+  - "Use good defaults everywhere possible."
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D49154%253A36853&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Breaking the long task into a series of smaller, linked steps, lets you:
+
+- Distribute the overall cognitive load
+- Help the user understand the process by breaking it into more understandable chunks
+- Guide the user through the process with minimal friction.
+- Allow decisions made in preceding steps to vary the workflow of subsequent steps 
+
+### Anatomy
+
+- Each **step** in a workflow contains a logical grouping of form elements needed to complete the overall task.
+- The **workflow header** includes the Culture Amp logo, step title, and exit link:
+    - The **step title** matches the step title in the workflow footer [progress stepper](/components/progress-stepper).
+    - The **exit link** in the top–right (in left-to-right languages) returns the user to the rest of the product. It may link to the page that launched the workflow or a known sensible place to send the user. The label typically shows that it will "save & close", but in certain circumstances may need to be stronger, such as "exit".
+- The **workflow footer** contains previous and next buttons (labeled "Previous" and "Next):
+    - On the first step, no "previous" button is shown.
+    - On the final step, the "next" button changes to a confirmation of final behavior, such as "Launch".
+
+### Behavior
+
+Workflows include:
+
+- Steps that are completed. May be gated/ungated and/or mandatory/optional.
+- Front-end validation of supplied data and selected options. Some of this may be achieved via affordances (such as preventing forward navigation until necessary information has been entered). Form validation will be expressed via Kaizen styles.
+- Indications of steps and progress (max 5–6 steps, each labeled by a single word taken from the step title)
```

Comment:
If we have more than 5-6 steps, should we not use the workflow? What’s the alternative?